git-config/tests/git_config/mod.rs

```diff
@@ -675,6 +676,13 @@ mod from_paths_tests {
         );
     }
 
+    #[ignore]
```

Comment:
@Byron fails when `matches`'s  `value: impl Into<&'a BStr>` (so `from_paths::Options.git_dir`) is an absolute path. What am I missing?

Reply:
You have found a shortcoming that I will fix one way or another. The issue is that thus far all matches are considered relative to a repository, and there is an optimization from `ignore` I think that treats absolute patterns (starting with a 'slash' as special and removes the leading slash. When matching against absolute values, even if it's the same as the pattern, they won't match anymore. `git` doesn't do that kind of processing and I will see if I can remove it as well.